batches.go

```diff
@@ -0,0 +1,111 @@
+package packngo
+
+import (
+	"fmt"
+)
+
+const batchBasePath = "/batches"
+
+// BatchService interface defines available batch methods
+type BatchService interface {
+	Get(batchID string, listOpt *ListOptions) (*Batch, *Response, error)
+	List(ProjectID string, listOpt *ListOptions) ([]Batch, *Response, error)
+	Create(projectID string, batches *InstanceBatchCreateRequest) (*Batch, *Response, error)
+}
+
+// Batch type
+type Batch struct {
+	ID                     string     `json:"id,omitempty"`
```

Comment:
I imagine some of the parameters will always be there - ID, state, Href; so I would drop the omitempty from most of them.

You probably need to manually verify which are always there are and which not. Or maybe you can see the api-docs json.